src/component/BarreDeRecherche.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+const axios = require('axios');
```

Comment:
@malvena15 Why make use of both `import` and `require`?

Reply:
I understand both and I can use both for. To be honest, I simply used copy and paste from the Axio git readme file. 

I know that require is the syntax for common javascript and Import it for ECMAScript 6. But if we use Babel import will be converting the require. So at the and it will be the same